pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -126,6 +127,8 @@ private PGStream tryConnect(Properties info, SocketFactory socketFactory, HostSp
     int connectTimeout = PGProperty.CONNECT_TIMEOUT.getInt(info) * 1000;
     String user = PGProperty.USER.getOrDefault(info);
     String database = PGProperty.PG_DBNAME.getOrDefault(info);
+    String sslNegotiation = PGProperty.SSL_NEGOTIATION.getOrDefault(info);
```

Comment:
WDYT of making an enum like we have for `org.postgresql.jdbc.PreferQueryMode`, `org.postgresql.jdbc.AutoSave`, `org.postgresql.jdbc.EscapeSyntaxCallMode`?